src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -144,7 +145,10 @@ def _list_devices(self, cmd):
         >
```

Comment:
Could you also remove the
```
        This (all lines starting with ">") is how it is supposed to work. As of
        now, it's not yet implemented:
```
line and the `> ` prefixes?

Reply:
Removed. Thx!